modules/grafana/manifests/dashboards/application_dashboard.pp

```diff
@@ -106,6 +108,16 @@
     $elasticsearch_stats = []
   }
 
+  if $show_memcached {
+    $memcached_row = [
+      [
+        'memcached_free',
+        'memcached_used']
```

Comment:
Have you got a preview of what this looks like? It seems a little odd to have two panels, that sound like they display connected data, as if more memory is used in memcached, then there will be less space free?

Reply:
I'm open to opinions on this. The original card wanted both, and trying to put them on the same graph (stacking) just shows a huge wall of one colour, due to having an extra 3 zeros I imagine. I imagine the free is more useful to display than the used, so I'm happy to remove one of them. 
I'll create the panel again quickly and add some screenshots.

Reply:
![screen shot 2019-02-26 at 12 14 17](https://user-images.githubusercontent.com/5594262/53412041-20e6a000-39c0-11e9-8f3c-d55d6cbf7e41.png)

Reply:
this is admittedly on int rather than prod, so the numbers are probably pretty different.

Reply:
I'd suggest trying a panel per machine, using the repeating panels feature. Also, have a look at the Machine dashboard, at how it displays memory usage. I think it's visualising similar data and uses the stacking and conditional formatting.

Reply:
cool, thanks, will check it out

Reply:
On reflection, I don't think its that valuable to show the stacked memory: the things we want to monitor is whether we're running out of memory, and whether we have spikes/changes. due to the scale stacking it would hide these changes to casual observers, and knowing the memory used and the memory free isn't any more useful than knowing just one, hence the decision to only have the single memcached graph